open-authoring/error-interpreter/client-side/index.ts

```diff
@@ -1,14 +1,19 @@
-import OpenAuthoringError from "../../OpenAuthoringError"
-import interpretUnauthorizedError from "./unauthorized"
-import interpretNotFoundError from "./notFound"
+import OpenAuthoringError from '../../OpenAuthoringError'
+import interpretUnauthorizedError from './unauthorized'
+import interpretNotFoundError from './notFound'
 
-export default async function interpretClientError(error: OpenAuthoringError) {
-    switch (error.code) {
-        case 401: { // Unauthorized
-            return interpretUnauthorizedError(error)
-        }
-        case 404: { // Not Found
-            return await interpretNotFoundError(error)
-        }
+export default async function interpretClientError(
+  error: OpenAuthoringError,
+  github: any
+) {
+  switch (error.code) {
+    case 401: {
+      // Unauthorized
+      return interpretUnauthorizedError(error)
     }
-}
\ No newline at end of file
+    case 404: {
+      // Not Found
+      return await interpretNotFoundError(error, github)
```

Comment:
this makes me feel like maybe something here should actually be in `cms.api.github`...but I'm not exactly sure how

Reply:
I haven't really touched any of the error-interpreter stuff in this, as I'm thinking it may be a bit too big of a step.